src/js/sdk.ts

```diff
@@ -63,7 +67,7 @@ export function init(
                 : `${appPrefix}/${frame.filename}`;
           }
           return frame;
-        },
```

Comment:
Is this a `.editorconfig` or else that automatically strips these when the file is saved? We'd ideally share the linter/formatter rules etc so these things don't change on each PR.

Reply:
I have prettier on my VSCode that formats on save, I actually made a config that made it leave trailing commas alone but I think I accidentally removed it on a recent rebase I did, my bad!